catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/AskBrickTest.java

```diff
@@ -0,0 +1,89 @@
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.uiespresso.content.brick;
+
+import org.catrobat.catroid.R;
+import org.catrobat.catroid.content.Script;
+import org.catrobat.catroid.content.bricks.AskBrick;
+import org.catrobat.catroid.ui.ScriptActivity;
+import org.catrobat.catroid.uiespresso.util.BaseActivityInstrumentationRule;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.typeText;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+import static org.catrobat.catroid.uiespresso.content.brick.BrickTestUtils.checkIfBrickAtPositionShowsString;
+import static org.catrobat.catroid.uiespresso.content.brick.BrickTestUtils.enterStringInFormulaTextFieldOnBrickAtPosition;
+import static org.catrobat.catroid.uiespresso.util.UiTestUtils.getResourcesString;
+
+public class AskBrickTest {
+	private int setBrickPosition;
+
+	@Rule
+	public BaseActivityInstrumentationRule<ScriptActivity> baseActivityTestRule = new
+			BaseActivityInstrumentationRule<>(ScriptActivity.class, true, false);
+
+	@Before
+	public void setUp() throws Exception {
+		setBrickPosition = 1;
+		Script script = BrickTestUtils.createProjectAndGetStartScript("askBricksTest");
+		script.addBrick(new AskBrick(getResourcesString(R.string.brick_ask_default_question)));
+		baseActivityTestRule.launchActivity(null);
+	}
+
+	@Test
+	public void testAskBrick() {
+		String question = "Will it work?";
+		checkIfBrickAtPositionShowsString(0, R.string.brick_when_started);
+		checkIfBrickAtPositionShowsString(setBrickPosition, R.string.brick_ask_label);
+		checkIfBrickAtPositionShowsString(setBrickPosition, R.string.brick_ask_store);
+		enterStringInFormulaTextFieldOnBrickAtPosition(question, R.id.brick_ask_question_edit_text, setBrickPosition);
+	}
+
+	@Test
+	public void testCreatingNewVariable() {
+		final String variableName = "testVariable";
+		final int intToChange = 28;
+		checkIfBrickAtPositionShowsString(0, R.string.brick_when_started);
+		checkIfBrickAtPositionShowsString(setBrickPosition, R.string.brick_ask_label);
+		checkIfBrickAtPositionShowsString(setBrickPosition, R.string.brick_ask_store);
+		onView(withId(R.id.brick_ask_spinner)).perform(click());
```

Comment:
please use utility functions to achieve that. (see createNewVariableOnSpinnerInitial and createNewVariableOnSpinner)